gbpservice/nfp/configurator/drivers/firewall/vyos/vyos_fw_driver.py

```diff
@@ -701,3 +701,24 @@ def delete_firewall(self, context, firewall, host):
             self._print_exception('Failure', resp.status_code, url,
                                   'create', resp.content)
             return const.STATUS_ERROR
+
+    def configure_healthmonitor(self, context, kwargs):
+        return self._check_vm_health(kwargs)
+
+    def _check_vm_health(self, kwargs):
```

Comment:
The code for _check_vm_health() is same in base driver and all vendor drivers. Can we have this method in base driver and pass command as additional argument?